File: samples_generated_code/sample_2/phi4/baseline_code_py/code_row_81.py
import sys

def getValueFromArray(arr, size, index):
    if 0 <= index < size:
        return arr[index]
    else:
        return -1

def main():
    try:
        n = int(input().strip())
        id_sequence = list(map(int, input().split()))
        
        result = getValueFromArray(id_sequence, len(id_sequence), n)
        
        sys.exit(result)
    except Exception as e:
        sys.exit(-1)

if __name__ == "__main__":
    main()

File: samples_generated_code/sample_2/qwen/baseline_code_java/code_row_81.java
import java.util.Scanner;

public class Main {
    public static int getValueFromArray(int[] array, int size, int index) {
        if (index >= 0 && index < size) {
            return array[index];
        } else {
            System.exit(1); // Exit with error code if index is out of bounds
            return -1; // This line is unreachable but added for completeness
        }
    }

    public static void main(String[] args) {
        int[] id_sequence = {10, 20, 30, 40, 50};
        Scanner scanner = new Scanner(System.in);
        int index = scanner.nextInt();
        scanner.close();

        int result = getValueFromArray(id_sequence, id_sequence.length, index);
        System.exit(result);
    }
}